src/ReactFinalForm.js

```diff
@@ -105,8 +105,8 @@ export default class ReactFinalForm extends React.PureComponent<Props, State> {
 
   notify = (state: FormState) => this.setState({ state })
 
-  handleSubmit = (event: SyntheticEvent<HTMLFormElement>) => {
-    event.preventDefault()
+  handleSubmit = (event?: SyntheticEvent<HTMLFormElement>) => {
```

Comment:
Not sure if it should be `event?: SyntheticEvent` or `event: ?SyntheticEvent`

Reply:
In this case, I think it does not matter, but I would choose the latter.

Reply:
I think since the parameter is optional it's good this way: https://flow.org/en/docs/types/functions/#optional-parameters-a-classtoc-idtoc-optional-parameters-hreftoc-optional-parametersa

Reply:
Okay. I stand corrected with citation! 👍